gflownet/gflownet.py

```diff
@@ -57,6 +53,7 @@ def __init__(
         sample_only=False,
         replay_sampling="permutation",
         train_sampling="permutation",
+        eval_config=None,
         **kwargs,
     ):
         # Seed
```

Comment:
Deprecated? Looks only used in the `Buffer`'s creation but `Buffer.__init__` does not use it at all

Reply:
I am actually confused about what this refers to. Is your question about `eval_config`?

Reply:
It has moved unfortunately... It was referring to `sample_only`

Reply:
Oh I see! Yeah `sample_only` is very much deprecated.

Reply:
Done in [`4beae50` (#287)](https://github.com/alexhernandezgarcia/gflownet/pull/287/commits/4beae5045db277d424a641da68454d1dc74e7cb6)